UDV-Core/src/Modules/Temporal/viz.js

```diff
@@ -0,0 +1,90 @@
+import * as vis from 'vis-network';
+
+/**
+* Manager for the graph
+* Done as a singleton so only one instance can exist.
+* If a constructor is called :
+*   - First time = instantiation
+*   - Other time = return the already existing NetworkManagerSingleton
+*/
+export class NetworkManagerSingleton {
+    /**
+     * Constructs a NetworkManagerSingleton.
+     *
+     * @param {String} id_network - HTML id which will be the container of the graph
+     * To simplify the use of the singleton, others parameter are not filled in the construction phases
+     *
+     * network {Object} - hold the network/graph instance created by viz.js
+     * data {Object} - Data about nodes, edges and groups for the graph
+     * option {Object} - Data about graphics' options for viz.js
```

Comment:
Same remark than for `data`, you need to be more specific on what we can put in option and when it is done since it is not done in the constructor.

Reply:
Comments added